ansible_runner/runner_config.py

```diff
@@ -242,6 +242,13 @@ def prepare_env(self):
         self.job_timeout = self.settings.get('job_timeout', None)
         self.pexpect_timeout = self.settings.get('pexpect_timeout', 5)
 
+        self.process_isolation = self.settings.get('process_isolation', False)
```

Comment:
The defaults in the documentation do no match the actual defaults we set in the code because those defaults are set by runner itself.